package.json

```diff
@@ -5,7 +5,7 @@
   "main": "prism.js",
   "style": "themes/prism.css",
   "scripts": {
-    "test": "mocha tests/testrunner-tests.js && mocha tests/run.js && mocha tests/plugins/**/*.js"
+    "test": "mocha tests/testrunner-tests.js && mocha tests/plugins/**/*.js && mocha tests/run.js"
```

Comment:
We should split this into 3 commands, and have `test` run them, so if someone wants to test the languages, they could do `npm run test:languages -- --language=css` without issue. It also allows you to e.g. just test plugins without needing to run the other tests around it.